WordPress/Classes/Services/NotificationFilteringService.swift

```diff
@@ -14,16 +14,16 @@ final class NotificationFilteringService {
     var wordPressNotificationsEnabled: Bool {
         get {
             guard let userDefaults = userDefaults,
-                  userDefaults.value(forKey: WPNotificationsEnabledKey) != nil else {
```

Comment:
There'll be conflicts with these after my latest merge.

WPNotificationsEnabledKey and subsequently AppConfiguration.Extension.NotificationsService.enabledKey are no longer needed and could be deleted.

Reply:
Thanks for the heads up @staskus 🙇